pkg/instancetype/instancetype_test.go

```diff
@@ -1484,6 +1487,33 @@ var _ = Describe("Instancetype and Preferences", func() {
 				}
 			})
 
+			Context("PreferredInterfaceMasquerade", func() {
+				It("should be applied to primary interface on Pod network", func() {
+					vmi.Spec.Networks = []v1.Network{{
+						Name: vmi.Spec.Domain.Devices.Interfaces[0].Name,
+						NetworkSource: v1.NetworkSource{
+							Pod: &v1.PodNetwork{},
+						},
+					}}
+					Expect(instancetypeMethods.ApplyToVmi(field, instancetypeSpec, preferenceSpec, &vmi.Spec)).To(BeNil())
+					Expect(vmi.Spec.Domain.Devices.Interfaces[0].Masquerade).ToNot(BeNil())
+					Expect(vmi.Spec.Domain.Devices.Interfaces[1].Masquerade).To(BeNil())
+				})
+				It("should not be applied if primary interface has another binding set", func() {
+					vmi.Spec.Domain.Devices.Interfaces[0].SRIOV = &v1.InterfaceSRIOV{}
+					Expect(instancetypeMethods.ApplyToVmi(field, instancetypeSpec, preferenceSpec, &vmi.Spec)).To(BeNil())
+					Expect(vmi.Spec.Domain.Devices.Interfaces[0].Masquerade).To(BeNil())
+					Expect(vmi.Spec.Domain.Devices.Interfaces[0].SRIOV).ToNot(BeNil())
+				})
+				It("should not be applied if primary interface is not on Pod network", func() {
```

Comment:
I think the terms primary and pod interface refer to the same, I would rephrase to "should not be applied on interface that is not on Pod network"